Tests/iaas/flavor-naming/check_yaml.py

```diff
@@ -8,9 +8,9 @@
 
 Also check that the flavor properties encoded in the YAML match those encoded in the name.
 """
-import os
-import os.path
+
 import sys
+from pathlib import Path
```

Comment:
I would sort imports by the module, regardsless of `import` vs `from ... import`. No?